src/Time/Units.hs

```diff
@@ -44,6 +44,7 @@ module Time.Units
        , floorUnit
```

Comment:
Oh, and please add an "Unreleased" section to the changelog, and an entry under that explaining what changed and how to migrate.